fms2_io/netcdf_io.F90

```diff
@@ -1621,11 +1615,22 @@ elemental function is_valid(datum, validobj) &
 
   valid_data = .true.
   if (validobj%has_range) then
-    valid_data = rdatum .ge. validobj%min_val .and. rdatum .le. validobj%max_val
-  elseif (validobj%has_fill) then
-    valid_data = rdatum .ne. validobj%fill_val
-  elseif (validobj%has_missing) then 
-    valid_data = rdatum .ne. validobj%missing_val
+    if (validobj%has_min .and. .not. validobj%has_max) then
```

Comment:
Is this possible with the new code?  If there is a range, don't you set these flags both to `.true.`

Reply:
As per my comment from the get_valid call, in the open range definition of `valid%has_range` this is a possibility (and potentially a frequent one with _FillValue defining a min or max)... obviously if we want to set `valid%has_range` to the closed definition I'd have to rework this section (and the others).